coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 98.6
+    "covered_percent": 98.48
```

Comment:
Hmm, so are you introducing untested code?

Reply:
shouldn't be, i am not certain how this is calculated but perhaps the total number of lines has increased

Reply:
looks like there might be a couple lines that were not tested